packages/freezeframe/src/index.ts

```diff
@@ -124,8 +124,12 @@ class Freezeframe {
   private _process(freeze: Freeze): Promise<Freeze> {
     return new Promise((resolve) => {
       const { $canvas, $image, $container } = freeze;
-      const { clientWidth, clientHeight } = $image;
+      const { width, height } = $image.getClientRects()[0];
+      const clientWidth = Math.ceil(width);
+      const clientHeight = Math.ceil(height);
```

Comment:
clientWidth and clientHeight stay, but are now a ceiled version of the "real" dimensions of the image.